prodsys/simulation/request.py

```diff
@@ -80,18 +80,35 @@ class TransportResquest(Request):
     """
     def __init__(
         self,
-        process: process.TransportProcess,
+        process: Union[process.TransportProcess, process.LinkTransportProcess],
         product: product.Product,
         origin: product.Location,
         target: product.Location,
+        path = None,
```

Comment:
I think this is really inconvenient -> the transport request should hold information about the process to perform, but why here information about the path to take. All of this should be contained in the controller...

Reply:
Commit 32: Possible to delete path to origin. Path__to_target needs to be added to each Transportrequest in the process.py and will then be selected by heuristic with resource in the router.py.